### PR TITLE
fix: scatter plot tooltip issue and remove cloudinary rounded image solution

### LINKED ISSUE #780: Bug: contributor hover-card is not clickable
### Describe the bug

The contributor hover-card is not clickable when tried to click in the dashboard view
image

### Steps to reproduce

1. Go to https://insights.opensauced.pizza/javascript/dashboard/filter/recent
2. Click on the dashboard view
3. hover on a contributor and try clicking on the contributor hover card.
4. contributor hover card cannot be clicked as the card disappears
recording

### Affected services

insights.opensauced.pizza

### Platforms

Desktop

### Browsers

Chrome

### Environment

Production

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by OgDev-01**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [x] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
This PR:

- Updates the rendered svg node on the scatterplot
- removes all instances of Cloudinary rounded image helper usage
- partially fixes scatterplot tooltip hover issue we had open in https://github.com/plouc/nivo/issues/2201

### Actual problem
- Not being able to render and style images to be rounded (we got around this using the Cloudinary rounded mage transformation utils function)
- We could not hover on the tooltip provided by `nivo` on the `scatterplot` of which we opened an issue for [here](https://github.com/plouc/nivo/issues/2201)

### Solution
I was working on a personal project and ran into svgs... and research lead me to find out about [foreignObject](https://www.geeksforgeeks.org/svg-foreignobject-element/) in svgs. With the `foreignObject` element, I could render none ` Scalable Vector Graphic` elements as svg. And that was the actual solution to all the above-mentioned issues we had.

## Related Tickets & Documents
Fixes #780

### Before

https://github.com/open-sauced/insights/assets/62995161/50955508-74b7-42b0-b202-3ca6b2b5f95f

### After
https://github.com/open-sauced/insights/assets/62995161/04509209-d772-4756-aa64-fa2864b165da

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: components/atoms/Avatar/avatar-hover-card.tsx [hunk 1]
```diff
@@ -5,37 +5,32 @@ import Link from "next/link";
 import HoverCardWrapper from "components/molecules/HoverCardWrapper/hover-card-wrapper";
 
 import { getAvatarByUsername } from "lib/utils/github";
-import roundedImage from "lib/utils/roundedImages";
```
Comment (bdougie) [inline components/atoms/Avatar/avatar-hover-card.tsx:8]: Is this no longer being used in the app? Are we still caching using cloudinary?

**Comment by bdougie** [comment]
This functionally works, great job! 

Can you provide more context to your approach for this fix?
- Is the `lib/utils/roundedImages` still needed at this point and should that be deleted? 
- Are we still leveraging cloudinary for caching. I don't remember how much of that was baked into the rounded images.

File: components/atoms/Avatar/avatar-hover-card.tsx (hunk 1 above)
Comment (OgDev-01) [reply components/atoms/Avatar/avatar-hover-card.tsx:8]: Yes!, the cloudinary rounded image transformation helper is no longer in use.

**Comment by OgDev-01** [comment]
> This functionally works, great job!
> 
> Can you provide more context to your approach for this fix?
> 
> * Is the `lib/utils/roundedImages` still needed at this point and should that be deleted?
> * Can you explain the SVG approach to this fix a bit more? I am not quite following the context from just looking at the code.
> * ~Are we still leveraging cloudinary for caching? I don't remember how much of that was baked into the rounded images.~
> 
> update:
> 
> I already see we are caching separately
> 
> https://github.com/open-sauced/insights/blob/beta/lib/utils/cachedImages.ts

Updates: I have added the pr Description with the actual problem and solution approach 
cc: @bdougie

**Review by brandonroberts — APPROVED** [review]

**Comment by OgDev-01** [comment]
> @OgDev-01 are you going to delete the `lib/utils/roundedImages.ts` file also if its not being used?

Ooh, i missed that...deleting shortly :<

**Pull request merged**